pulseapi/entries/views.py

```diff
@@ -328,6 +328,8 @@ class EntriesListView(ListCreateAPIView):
         'description',
         'get_involved',
         'interest',
+        'related_creators__creator__name',
+        'published_by__name',
```

Comment:
hmm, I wonder if it would be better for this to be `published_by__profile__name` (not sure if it would work since `name` isn't an actual model field but is a property on the class)